src/components/ConfirmDialog.react.js

```diff
@@ -99,7 +99,7 @@ ConfirmDialog.propTypes = {
     /**
      *  Set to true to send the ConfirmDialog.
      */
-    displayed: PropTypes.bool,
+    displayed: PropTypes.oneOfType([PropTypes.oneOf([null]), PropTypes.bool]),
```

Comment:
Necessary because of [this](https://github.com/plotly/dash-core-components/blob/b7c2e4a3d35b53e0e6bd34f25610319fe50804f1/test/test_integration.py#L1059-L1063) test, since it will set `displayed=None`. Or, should we change _that_ test and keep this as only `PropTypes.bool`